cve_bin_tool/checkers/libxkbcommon.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/python3
+
+"""
+CVE checker for libxkbcommon
+
+References:
+https://www.cvedetails.com/vulnerability-list/vendor_id-19208/product_id-49883/version_id-259877/Xkbcommon-Libxkbcommon-0.8.1.html
+
+"""
+
+from . import Checker
+
+
+class LibxkbcommonChecker(Checker):
+	CONTAINS_PATTERNS = []
+	FILENAME_PATTERNS = [r"libxkbcommon.so", r"libxkbcommon-x11.so"]
```

Comment:
`[r"libxkbcommon.so", r"libxkbcommon-x11.so"]` could be changed to `[r"libxkbcommon"]`. Which one would be better?

Reply:
In this case, it's probably clearer to have them separate to show that there's two potential files being detected.  If want to be fancy `r"libxkbcommon(-x11)?\.so"`covers both filenames in a single pattern, or just leave a comment indicating that there's two library files you want to match.